docs/contributing/docs.md

```diff
@@ -2,186 +2,504 @@
+>personal appearance, body size, race, ethnicity, age, religion, or nationality.
+<!-- vale on -->
 
-## Licensing
-The Crossplane documentation is under the [Creative Commons Attribution](https://creativecommons.org/licenses/by/4.0/) license. CC-BY allows reuse, remixing and republishing of Crossplane documentation with attribution to the Crossplane organization.
+### Reporting violations
+To report violations contact the Crossplane maintainers at `info@crossplane.io`
+or the CNCF at `conduct@cncf.io`.
 
+## Docs source 
+Crossplane documentation lives in two repositories:
 
-### Directory structure
-The relevant directories of the [docs repository](https://github.com/crossplane/crossplane.github.io) are:
-| Directory | Purpose |
-| ----- | ---- |
-| `content` | Markdown files for all individual pages. |
-| `static/images` | Location for all image files.  |
-| `themes` | HTML templates and Hugo tooling. |
+* Crossplane documentation source is in the [Crossplane
+repository](https://github.com/crossplane/crossplane) `/docs` directory. 
 
-All markdown content exists within the `/content` directory.   
-All image files exist within `static/images`.
+* The Crossplane docs website is in the
+  [crossplane.github.io](https://github.com/crossplane/crossplane.github.io)
+repository.
 
-### Run Hugo
-* Follow the [Hugo documentation](https://gohugo.io/getting-started/installing/) to install Hugo.
-* Run `hugo server`
-* Go to [http://localhost:1313](http://localhost:1313)
+Use `crossplane/crossplane` for documentation contributions.  
+Use `crossplane/crossplane.github.io` for local development or updates involving
+HTML, CSS or Hugo.
 
-Crossplane documentation uses [Hugo v0.101.0](https://github.com/gohugoio/hugo/releases/tag/v0.101.0), but newer versions are compatible.
+### Licensing
+The Crossplane documentation is under the [Creative Commons
+Attribution](https://creativecommons.org/licenses/by/4.0/) license. CC-BY allows
+reuse, remixing and republishing of Crossplane documentation with attribution to
+the Crossplane organization.
 
-### Documentation issues and feature requests
-Use [GitHub Issues](https://github.com/crossplane/crossplane.github.io/issues) to report a problem or request documentation content.
+### Issues and feature requests
+Open an [issue](https://github.com/crossplane/crossplane/issues)
+to report a problem or request documentation content.
 
 ## Contributing
 
-The `/content` directory contains all documentation pages.  
-Each version of Crossplane software maintains a unique directory within `content`.
+Documentation content contributions are always welcome.
 
-### Front matter
-Each page contains metadata called [front matter](https://gohugo.io/content-management/front-matter/). Each page requires front matter to render.
+The Crossplane documentation source is in the [Crossplane
+repository](https://github.com/crossplane/crossplane) `/docs` directory. 
```

Comment:
This appears to be repeated above.

Reply:
This is intentional to try and avoid a mix up between the two repos.

Reply:
ah i see -- this is also slightly different from what is said above 👍🏻